spec-main/chromium-spec.ts

```diff
@@ -367,6 +367,36 @@ describe('web security', () => {
   });
 });
 
+ifdescribe(process.platform === 'linux')('subprocesses', () => {
+  let appProcess: ChildProcess.ChildProcessWithoutNullStreams | undefined;
+  afterEach(() => {
+    if (appProcess && !appProcess.killed) {
+      appProcess.kill();
+      appProcess = undefined;
+    }
+  });
+
+  it('does not propagate GDK_BACKEND', async () => {
+    const appPath = path.join(fixturesPath, 'api', 'gdk-backend-check');
+    appProcess = ChildProcess.spawn(process.execPath, [appPath]);
```

Comment:
The child process should have `GDK_BACKEND` explicitly cleared in its environment variables.

Also we can probably add a test to explicitly set the `GDK_BACKEND` and check the result.